gremlin-language/src/main/antlr4/Gremlin.g4

```diff
@@ -1928,18 +1956,22 @@ K_ANY: 'any';
 K_AS: 'as';
 K_ASC: 'asc';
 K_ASDATE: 'asDate';
+K_ASNUMBER: 'asNumber';
 K_ASSTRING: 'asString';
 K_ASSIGN: 'assign';
 K_BARRIER: 'barrier';
 K_BARRIERU: 'Barrier';
 K_BEGIN: 'begin';
 K_BETWEEN: 'between';
+K_BIGDECIMAL: 'nbigDecimal';
```

Comment:
I'm not sure if we want to let the java keyword avoidance leak into the grammar here with the `n` prefixes. The pros and cons of consistency amongst language variants vs limiting JVM pollution of the language should be weighed independently of the Java enum debate.